docs/quickstarts/python-quickstart.md

```diff
@@ -0,0 +1,203 @@
+---
+layout: docs
+title: Python Quickstart
+---
+
+<h1 class="page-header">Python Quickstart</h1>
+
+<p class="lead">This guide gets you started with gRPC in Python with a simple working example.</p>
+
+<div id="toc"></div>
+
+## Before you begin
+
+### Prerequisites
+
+[versions of pip, any other tools, etc. go here]
+
+If you haven't done so already, install the gRPC runtime and gRPC tools.
+
+### Install gRPC
+
+Install gRPC:
+
+```sh
+  $ pip install grpcio
+```
+
+Or, to install it system wide:
+
+```sh
+  $ sudo pip install grpcio
+```
+
+If you're on Windows, make sure you installed the `pip.exe` component when you
+installed Python. Invoke as above but with `pip.exe` instead of `pip` (you may
+also need to invoke from a `cmd.exe` ran as administrator):
+
+```sh
+  $ pip.exe install grpcio
+```
+
+### Install gRPC tools
+
+Python's gRPC tools include the protocol buffer compiler `protoc` and the special plugin for generating server and client code from `.proto` gRPC service definitions. For the first part of our quickstart example, we've already generated the server and client stubs from [helloworld.proto](https://github.com/grpc/grpc/tree/{{ site.data.config.grpc_release_branch }}/examples/protos/helloworld.proto), but you'll need the tools for the rest of our quickstart, as well as later tutorials and your own projects.
```

Comment:
I suspect this should be `generating gRPC server and client code from .proto service definitions`. Nominally `.proto` files and protocol buffers in the large "don't know" about gRPC; they just offer a generic plug-in system of which gRPC makes use.